enhancements/metering/user-provided-pricesheets.md

```diff
@@ -0,0 +1,205 @@
+---
+title: user-provided-price-sheets
+authors:
+  - "@chancez"
+reviewers:
+  - @tflannag
+  - "@emilym1"
+approvers:
+  - "@operator-framework/team-metering"
+creation-date: 2019-11-06
+last-updated: 2019-11-06
+status: implementable
+---
+
+# user-provided-price-sheets
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Open Questions [optional]
+
+TBD
+
+## Summary
+
+User provided price sheets is a way for users to define custom price rates for calculating cost reports on resource reservations or usage.
+It is an opt-in feature that would require the user to provide configuration that defines the prices and rates they desire.
+
+## Motivation
+
+Currently metering only supports calculating costs using AWS billing data, but does not support calculating costs for other clouds nor on-premise solutions.
+User provided price sheets would allow users to define their own prices for uses in clouds or environments which don't yet have native support from Metering.
+
+Additionally, by creating the concept of price sheets, we must define our cost report queries to work against a well defined API.
+Instead of writing a cost report query per cloud provider, we can write queries that will work against any price sheet.
+This would allow us, and users to programmatically generate price sheets, more easily unlocking support cost reports on different clouds and on-premise environments.
+
+### Goals
+
+- Allow users to define their own prices for resources, eg: $.015 per core hour.
```

Comment:
$0.15/core hour?

Reply:
I actually meant what i put but there should be a leading 0 before the decimal place, eg `$0.015`. `$0.15` is really expensive.

Reply:
That's my mistake, I meant deflating the "per hour" into a slash -> "$0.015/core hour"

Reply:
I kinda like writing it out to be explicit but I could be convinced otherwise.